x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/action/LookupJoinTypesIT.java

```diff
@@ -361,16 +365,18 @@ private void initIndexes(String group) {
     private void initData(String group) {
         Collection<TestConfig> configs = testConfigurations.get(group).configs.values();
         int docId = 0;
+        List<IndexRequestBuilder> indexRequests = new ArrayList<>(configs.size());
         for (TestConfig config : configs) {
             String doc = String.format(Locale.ROOT, """
                 {
                   %s,
                   "other": "value"
                 }
                 """, lookupPropertyFor(config));
-            index(config.indexName(), "" + (++docId), doc);
-            refresh(config.indexName());
+            var indexRequest = client().prepareIndex().setIndex(config.indexName()).setId("" + (++docId)).setSource(doc, XContentType.JSON);
+            indexRequests.add(indexRequest);
         }
+        indexRandom(true, indexRequests);
```

Comment:
:+1:

If we were making more than one value per index this would speed things up a ton. As is it's probably pretty small, but still.